app/views/users/show.html.haml

```diff
@@ -4,24 +4,31 @@
   .wrapper
     %header.header Your calories are #{@calories}
     %aside.aside.aside-l
-      %h3 Breakfast - #{@breakTitle}
+      %h3 Breakfast: 
+      =link_to(@breakTitle, @breakLink)
+      
       %h5 Calories: #{@breakCals}
       %h5 Preparation time: #{@breakTime} mins
-      %img{:src => @breakImg, :width => "350", :height => "250"}
+
+      %img{:src => @breakImg, :width => "300", :height => "200"}
       = link_to 'Favorite Breakfast', favorite_recipe_path(:Type => "Breakfast", :Title => @breakTitle, :Calories => @breakCals, :PrepTime => @breakTime)
       
+      
     %aside.aside.aside-d
-      %h3 Lunch - #{@lunchTitle}
+      %h3 Lunch:
+      =link_to(@lunchTitle, @lunchLink)
       %h5 Calories: #{@lunchCals}
       %h5 Preparation time: #{@lunchTime} mins
-      %img{:src => @lunchImg, :width => "350", :height => "250"}
+      
+      %img{:src => @lunchImg, :width => "300", :height => "200"}
       = link_to 'Favorite Lunch', favorite_recipe_path(:Type => "Lunch", :Title => @lunchTitle, :Calories => @lunchCals, :PrepTime => @lunchTime)
 
     %aside.aside.aside-b
-      %h3 Dinner - #{@dinTitle}
+      %h3 Dinner:
+      =link_to(@dinTitle, @dinLink)
```

Comment:
It looks like you managed to update the linking across the entire app, well done